zerver/webhooks/flock/tests.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+from typing import Text
+from zerver.lib.test_classes import WebhookTestCase
+
+class FlockHookTests(WebhookTestCase):
+    STREAM_NAME = 'test'
+    URL_TEMPLATE = u"/api/v1/external/flock?api_key={api_key}"
+
+    def test_flock_message(self) -> None:
+        expected_subject = u"Flock notifications"
```

Comment:
Is there a way for the topic to be the channel name instead of just "Flock notifications"?

Reply:
@eeshangarg  Hmm...I think it's possible. There's a field `to` in fixtures which represent `channel_id`, we have to make a GET request to `list_channel` endpoint and match if payload's channel_id matches with any channel_id from the response. If matches, we supply it to the `topic` variable. Would it make sense?

Reply:
@Rishabh570: No that's not really good then! We don't want to have to interface with another API just to get the topic. Just leave it as it is! :)

Reply:
Okay :)